File: packages/create-sst/bin/presets/examples/rest-api-go/templates/db/notes.go
package db

import (
	"strconv"
	"time"
)

func Notes() map[string]map[string]string {
	return map[string]map[string]string{
		"id1": {
			"noteId":    "id1",
			"userId":    "user1",
			"content":   "Hello World!",
			"createdAt": strconv.FormatInt(time.Now().Unix(), 10),
		},
		"id2": {
			"noteId":    "id2",
			"userId":    "user2",
			"content":   "Hello Old World!",
			"createdAt": strconv.FormatInt(time.Now().Unix()-1000, 10),
		},
	}
}

File: packages/create-sst/bin/presets/examples/rest-api-python/templates/services/db/notes.py
import time
import numpy

def getNotes():
  return {
      "id1": {
          "noteId":    "id1",
          "userId":    "user1",
          "content":   str(numpy.array([1,2,3,4])),
          "createdAt": int(time.time()),
      },
      "id2": {
          "noteId":    "id2",
          "userId":    "user2",
          "content":   str(numpy.array([5,6,7,8])),
          "createdAt": int(time.time()-1000),
      },
  }
